vowelCount.js

```diff
@@ -0,0 +1,19 @@
+/*
+Return the number (count) of vowels in the given string.
+
+We will consider a, e, i, o, u as vowels for this Kata (but not y).
+
+The input string will only consist of lower case letters and/or spaces.
+*/
+
+function getCount(str) {
+  let arrayFromString = str.split("");
+  let result = 0;
+  
+  arrayFromString.forEach(letter => {
+    if(letter === 'a' || letter === 'e' || letter === 'i' || letter === 'o' || letter === 'u') {
```

Comment:
```js
const array1 = [1, 2, 3];

console.log(array1.includes(2));
```
Generalnie jak w kodzie się dużo powtarza to może znaczyć że jest jakiś prostrzy sposób.
Gdybyś chciał teraz liczyć spółgłoski, albo znaki specjalne ten kod byłby bardzo nieczytelny szybko i długi

Reply:
No w sumie racja. Zrobiłem w ten sposób - będzie git?